x-pack/plugins/security_solution/common/detection_engine/schemas/request/perform_bulk_action_schema.test.ts

```diff
@@ -41,12 +43,11 @@ describe('perform_bulk_action_schema', () => {
     const payload: Omit<PerformBulkActionSchema, 'action'> = {
       query: 'name: test',
     };
+    const message = retrieveValidationMessage(payload as PerformBulkActionSchema);
 
-    const decoded = performBulkActionSchema.decode(payload);
-    const checked = exactCheck(payload, decoded);
-    const message = foldLeftRight(checked);
     expect(getPaths(left(message.errors))).toEqual([
       'Invalid value "undefined" supplied to "action"',
+      'Invalid value "undefined" supplied to "updates"',
```

Comment:
Why did this message change? I think the previous variant was more accurate for this test case.